<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>820</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Liberation Serif&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_nInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>671</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Nadelinformationen</string>
   </property>
   <widget class="QLabel" name="label_nTip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>349</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>Nadelspitze:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_nDiameter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>keine Auswahl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FineNeedle: Gauge 30-33</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>StandardNeedle: Gauge 25-27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SutureNeedle: Gauge 18-22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SpinalNeedle: Gauge 16-25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biopsy Cutting Needle: Gauge 14-20</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_nPatient">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>294</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>Eindringtiefe der Nadel (mm) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nDiameter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>349</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>Nadelstärke:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_nTip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>keine Auswahl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Straight Needle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RoundBody Needle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Half-Curved Needle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle Needle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reverse Cutting Needle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Taper Point Needle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Atraumatic Needle</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_nLength">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>294</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>Nadel-Länge (mm):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wLocation">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>349</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>Wundposition:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wDepth">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>349</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>Wundtiefe (mm):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_wLocation">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>keine Auswahl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kopf/Hals</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brust/Bauch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rücken</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bein</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hand</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fuß</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Leiste</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gesäß</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gesicht</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_nStitches">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>294</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_nStitches">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>349</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl der Stichpunkte:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wSize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>349</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>Wundgröße (mm):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nLengh">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nPatientDepth">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_wDepth">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_wSize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>730</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>Weiter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_backfirst">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>730</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>Züruck</string>
   </property>
  </widget>
  <widget class="QLabel" name="flabel_nDiameter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>© 2023 AS Interactive HealthTech. Alle Rechte vorbehalten.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="flabel_heading_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>20</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Liberation Serif&quot;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#21a1c0;&quot;&gt;Erfassung chirurgischer Daten&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
